src/suparnaNJune20/DuplicateNoFromArray.java

```diff
@@ -0,0 +1,45 @@
+/*Assignment 20 : 28th Jun 2020
+ * Program 2: Find duplicate number on Integer array in Java?
+Hint: There is exactly one number is repeated in the array. You need to write a program to
+find that duplicate number. For example, if an array with length 6 contains numbers {0, 3,
+1, 2, 3}, then duplicated number is 3.*/
+package suparnaNJune20;
+
+public class DuplicateNoFromArray {
+
+	void displayArray(int[] arrayInt) {
+		System.out.println("Input:");
+		System.out.print(+arrayInt[0]);
```

Comment:
No need to write line 12 separately,  as we can start 'for loop' with index = 0

Reply:
changes done.